Item.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refrigerator_EX
+{
+    class Item
+    {
+        private static int nextID = 1;
+
+        private int ID;
+        private int shelfId;
+        private string name;
+        private string type;
+        private string kashroot;
+        private DateTime expiryDate;
+        private double spaceNeed;
+
+
+        public int Id
+        {
+            get { return ID; }
+            set {; }
+        }
+
+
+        public int ShelfId
+        {
+            get { return shelfId; } 
+            set {shelfId=value; } 
+        }
+
+
+        public string Name
+        {
+            get { return name; }
+            set {
+                if (!(value.Equals("")))
+                    name = value;
+                else
+                    Console.WriteLine("name must be contain string");
```

Comment:
raise exception in this case, console is not a way to communicate errors to the caller